lite/examples/image_segmentation/android/.gitignore

```diff
@@ -0,0 +1,7 @@
+.gradle
```

Comment:
Everything item in this file is already included below. Please remove this file.
https://github.com/tensorflow/examples/blob/master/lite/.gitignore

Reply:
Deleted.